packages/next/build/index.ts

```diff
@@ -2726,14 +2726,20 @@ export default async function build(
         pathname: makeRe(p.pathname ?? '**').source,
       }))
 
-      await promises.writeFile(
-        path.join(distDir, IMAGES_MANIFEST),
-        JSON.stringify({
-          version: 1,
-          images,
-        }),
-        'utf8'
-      )
+      if (images.unoptimized) {
+        // Skip writing the images manifest file
+        // so that any downstream consumer will know
+        // to disable the Image Optimization API
```

Comment:
@ijjk Curious about your thoughts here if we should solve this problem here or if we should assume that any downstream consumer will know to read the `images.unoptimized` config.

Reply:
I reverted this part since it is not necessary